src/Apps/MyCollection/Routes/EditArtwork/MyCollectionCreateArtwork.tsx

```diff
@@ -29,6 +38,51 @@ export const MyCollectionCreateArtwork: React.FC = () => {
     null
   )
 
+  const initialStep = enableNewMyCUploadFlow ? "artist-select" : "details"
+
+  const [currentStep, setCurrentStep] = useState<Step>(initialStep)
```

Comment:
Saving the current step in a simple state here because we don't need to access the step anywhere else for now. Open for suggestions to change the logic.